Controls/CkEditorMinimal/src/DotVVM.Contrib/Scripts/DotVVM.Contrib.CkEditorMinimal.js

```diff
@@ -0,0 +1,23 @@
+﻿
+function setupCkEditor(id) {
+    var editor = CKEDITOR.instances[id];
+    if (editor) {
+        editor.destroy(true);
+    }
+    CKEDITOR.replace(id);
+    CKEDITOR.add
+}
+
+function changeCkEditor(id, prop) {
+    CKEDITOR.instances[id].on('change', function () { prop.html(CKEDITOR.instances[id].getData()); });
+}
+
+ko.bindingHandlers["dotvvm-contrib-CkEditorMinimal"] = {
+    init: function (element, valueAccessor, allBindingsAccessor, viewModel, bindingContext) {
+        var prop = valueAccessor();
+        var elementId = element.id;
+
+        setupCkEditor(elementId);
+        changeCkEditor(elementId, prop);
```

Comment:
The binding handler does not seem to support ViewModel -> CKEditor binding. I have no idea how hard would it be to implement, but it would be at least worth mentioning in the docs.